_posts/2022-12-05-gitsign-cache.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "gitsign cache"
```

Comment:
This title will appear on the front page, so probably ought to be something like "How to Configure a Gitsign Cache" or some such.

Reply:
Ah I did not think about that, I'll update shortly. Thanks!